### PR TITLE
good-eggs-2

### PR DESCRIPTION
**Pull request by mgasner**

### CODE REVIEW

**Review by dwallace0723 — APPROVED** [review]
so good!

File: pages/blog/good-eggs-2.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+status: 'published'
+title: 'Good Data at Good Eggs: Data observability with the asset catalog'
+excerpt: 'What we’re aiming for with Dagster is a completely horizontal view of our data assets. Our analysts will be able to look up when a raw data ingest from Stitch occurred, when a dbt model ran, or when a plot was generated by a Jupyter notebook and posted in Slack, through a single portal — a single "pane of glass."'
+date: '2020-11-05'
+authors: ['davidjwallace']
+---
+
+_This is the third in a series of posts about putting Dagster into production at Good Eggs. Read the [introduction](/blog/good-eggs-0) and [part one](/blog/good-eggs-1) to learn more about our experience transforming our data platform with Dagster._
+
+Though Dagster isn’t a specialized data observability solution, one thing we found attractive about the platform is that it integrates support for the most basic, and most important, kind of observability.
```
Comment (schrockn) [inline pages/blog/good-eggs-2.mdx:12]: The commas around "and most imporant" make this hard to parse. I think it needs to be:

"Though Dagster isn’t a specialized data observability solution, one thing we found attractive about the platform is that it integrates support for the most basic and most important kind of observability."

OR

"Though Dagster isn’t a specialized data observability solution, one thing we found attractive about the platform is that it integrates support for the most basic -- and most important -- kind of observability."

File: pages/blog/good-eggs-2.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+What if an analyst notices that a dbt model is starting to feel a little sluggish? How easy is it for them to inspect the trend of its execution time?
+
+Every organization that works with data faces questions like these. Routing them all through the Platform Team just isn’t sustainable. Clients of the data platform need to be able to answer their questions on a self-service basis.
+
+## Self-service observability
+
+Although things like tables in a data warehouse, Jupyter notebook figures, and Mode Analytics reports are semantically very different, we can think of them all as [_assets_](https://docs.dagster.io/overview/asset-materializations) produced by the computations we do in our pipelines. In fact, one way to think about a data application is that the assets it produces are the interface to the system.
+
+Since Dagster is responsible for orchestrating the production of these assets anyway, it makes a lot of sense for it to put a common index on those assets. So Dagster includes a built in asset catalog, and Dagster solids can yield specialized structured metadata to indicate they have produced assets that should be placed in the catalog.
```
Comment (schrockn) [inline pages/blog/good-eggs-2.mdx:32]: built-in

File: pages/blog/good-eggs-2.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        metadata_entries=entries,
+        asset_key=node_result.node["unique_id"],
+    )
+```
+
+This surfaces details about the model run in the Dagster UI. Now, our analysts can use a single portal to look up the history of their tables themselves, without the platform team getting involved.
+
+<p align="center">
+  <img
+    src="/posts/good-eggs-2/search_dbt_model_asset.gif"
```
Comment (schrockn) [inline pages/blog/good-eggs-2.mdx:67]: can't see this image in preview btw

Comment (mgasner) [reply pages/blog/good-eggs-2.mdx:67]: hmm, it might be too big and you have an ad blocker or sth.. let me see
